v2/samples/kusto/v1api20240413/v1api20240413_principalassignment.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kusto.azure.com/v1api20240413
+kind: PrincipalAssignment
+metadata:
+  name: aso-sample-principalassignment
+  namespace: default
+spec:
+  owner:
+    name: aso-sample-database
+  principalId: "james.judd@aveva.com"
+  principalType: "User"
+  role: "Viewer"
+  tenantId: "d1fe8e44-4d35-4dad-8ea9-d9d9cb9a988c"
```

Comment:
With the use of `$importedConfigMapMode` as detailed in a separate comment, you can avoid putting _your_ `tenantId` in this sample.